src/main/java/picard/sam/SamToFastq.java

```diff
@@ -64,13 +49,13 @@
  * the read's sequence from input sam file will be reverse-complemented prior to writing it to FASTQ in order restore correctly
  * the original read sequence as it was generated by the sequencer.</p>
  * <br />
- " <h4>Usage example:</h4>" +
- " <pre>" +
- " java -jar picard.jar SamToFastq \\<br />" +
- "     I=input.bam \\<br />" +
- "     FASTQ=output.fastq" +
- " </pre>" +
- " <hr />"
+ * " <h4>Usage example:</h4>" +

```

Comment:
Not sure why the double quotes are there, those were there before.

Reply:
Can you please remove them, from here and from the javadoc for `SamToFastqWithTags`? Even though it was written this way before, it is incorrect to have double quotes in this part of the javadoc. You also don't want the `+` symbols, the `\` should not be doubled, and the `<br/>` results in an extra newline (because it's used inside a `<pre>`, which already preserves newlines). Instead it should look like:
```
 * <h4>Usage example:</h4>
 *  <pre>
 *  java -jar picard.jar SamToFastq \
 *      I=input.bam \
 *      FASTQ=output.fastq
 *  </pre>
 * <hr />
```

It is easy in the IDE to render javadoc so you can see right away what the output will look like, and fix it there.

Reply:
👍  I didnt even try looking at it in the viewer